webpack.config.js

```diff
@@ -1,53 +1,52 @@
+/* eslint-disable no-useless-escape */
```

Comment:
I did test the boilerplate and it was not building. The issue was some how this file was converted it content as typescript. I recover its content to JS and the system is able to build again.